packages/page-accounts/src/modals/Create.tsx

```diff
@@ -1,27 +1,38 @@
 // Copyright 2017-2020 @polkadot/app-accounts authors & contributors
 // SPDX-License-Identifier: Apache-2.0
 
+import NewPasswordInput from '@polkadot/app-accounts/modals/PasswordInputNew';
```

Comment:
Please change the ordering back.

- first types
- then externals
- (in the same block) polkadot-js packages
- then locals